app/views/membership_applications/show.html.haml

```diff
@@ -8,3 +8,21 @@
   - if @membership_application.phone_number
     %p= @membership_application.phone_number
   %p= mail_to @membership_application.company_email, @membership_application.company_email
+  .wpcf7
+    = form_for @membership_application, url: membership_application_path do |f|
+      - if @membership_application.status.to_i == 2
+        - chosen = 2
+      - elsif @membership_application.status.to_i == 3
+        - chosen = 3
+      - else
+        - chosen = 1
```

Comment:
`status` is implemented as a string (for the time being), so the above assignment expressions would not seem to work.